app/views/api/features/create.js.erb

```diff
@@ -1,6 +1,9 @@
-var html = "<%= escape_javascript(render :partial => 'feature', :locals => {:feature => @feature}) %>";
 jQuery.colorbox.close();
-jQuery("table#features tbody").append(html);
-jQuery("table#features tr.notice").hide();
+
+document.querySelector('table#features tbody').innerHTML += '<%= escape_javascript(render partial: 'feature', locals: { feature: @feature }) %>';
+
+<% if @plan.issuer.features.size == 1 %>
```

Comment:
```suggestion
<% if @plan.issuer.features.any? %>
```

or `.size > 0`. I think `== 1` is too restrictive...

Reply:
Fair enough, server is unreliable for this evident client-side responsibility. But I don't like to run this line every time needlessly, so I pass the ball to CSS.

Reply:
I think it's a good solution!